tests/test_token.py

```diff
@@ -21,51 +23,55 @@ def mock_request(method, url, **kwargs):
 
 class TokenTests(unittest.TestCase, CustomAssertions):
 
+    def setUp(self):
+        Config._instance = None
+
     @classmethod
     def tearDownClass(cls):
         if os.path.exists("./temp"):
             os.remove("./temp")
+        
+        os.environ.pop("CONFIG_FILE", None)
+
 
     @patch("pyrefinebio.http.requests.request", side_effect=mock_request)
     def test_token_create(self, mock_request):
         result = pyrefinebio.Token.create_token("")
         self.assertEqual(result, token["id"])
-        mock_request.assert_called_with("POST", "https://api.refine.bio/v1/token/", params=None, data=None)
 
-    @patch("pyrefinebio.token.yaml.dump")
-    @patch("pyrefinebio.token.open")
+
+    @patch("pyrefinebio.config.yaml.dump")
+    @patch("pyrefinebio.config.open")
     def test_token_save(self, mock_open, mock_yaml):
+        os.environ["CONFIG_FILE"] = "test"
         mock_open.return_value.__enter__.return_value = "file"
 
-        pyrefinebio.Token.save_token("123456789", "test")
+        pyrefinebio.Token.save_token("123456789")
 
         mock_open.assert_called_with("test", "w")
         mock_yaml.assert_called_with({"token": "123456789"}, "file")
 
 
     def test_token_save_creates_file(self):
+        os.environ["CONFIG_FILE"] = "./temp"
+
         if os.path.exists("./temp"):
             os.remove("./temp")
 
-        self.assertFalse(os.path.exists("./temp"))
-
-        pyrefinebio.Token.save_token("123456789", "./temp")
+        pyrefinebio.Token.save_token("123456789")
 
         self.assertTrue(os.path.exists("./temp"))
 
 
-    @patch("pyrefinebio.token.os.path.exists")
-    @patch("pyrefinebio.token.yaml.full_load")
-    @patch("pyrefinebio.token.open")
-    def test_token_load(self, mock_open, mock_load, mock_exists):
+    @patch("pyrefinebio.config.os.path.exists")
+    @patch("pyrefinebio.config.yaml.full_load")
+    @patch("pyrefinebio.config.open")
+    def test_token_get(self, mock_open, mock_load, mock_exists):
+        os.environ["CONFIG_FILE"] = "./temp"
+        mock_open.return_value.__enter__.return_value = "file"
+        mock_load.return_value = {"token": "this-is-a-test-token"}
         mock_exists.return_value = True
-        mock_load.return_value = {"token": "123456789"}
-
-        token = pyrefinebio.Token.load_token("test")
 
-        mock_open.assert_called_with("test")
-        self.assertEqual(token, "123456789")
```

Comment:
Can we keep this assertion?

Reply:
Oh wait, I see that we assert below this, I thought for a sec that was a separate test.